testkit/tests/change_configuration.rs

```diff
@@ -72,6 +72,16 @@ fn test_configuration_and_rollbacks() {
     testkit.create_blocks_until(Height(10));
     assert_eq!(testkit.actual_configuration(), new_config);
 
+    testkit.checkpoint();
+    testkit.create_block();
+    testkit.rollback();
+    assert_eq!(testkit.actual_configuration(), new_config);
+
+    testkit.checkpoint();
```

Comment:
I don't quite understand why 2 instances of this code are required. IMO, it could make sense to setup one checkpoint immediately after a new config is committed:

```rust
testkit.commit_configuration_change(proposal);
testkit.checkpoint(); // <<< new code
testkit.create_blocks_until(Height(10));
assert_eq!(testkit.actual_configuration(), new_config);

testkit.rollback();
assert_eq!(testkit.actual_configuration(), old_config);
testkit.create_blocks_until(Height(10));
assert_eq!(testkit.actual_configuration(), new_config);

// other code...
```

Reply:
This is not going to go through because of how `commit_configuration_change` in testkit works:
1. `commit_configuration_change` schedules configuration for update in the next block.
2. block is created, configuration updated
If testkit rollbacks this block here - we get `old_config` (since `new_config` is no longer in blockchain). However, `testkit.create_blocks_until(Height(10));` won't change it to `new config`, because we don't have it scheduled by `commit_configuration_change`.